packages/table/src/quadrants/tableQuadrant.tsx

```diff
@@ -149,7 +150,7 @@ export class TableQuadrant extends AbstractComponent<ITableQuadrantProps, {}> {
                 >
                     <div className={Classes.TABLE_TOP_CONTAINER}>
                         {maybeMenu}
-                        {this.props.renderColumnHeader(showFrozenColumnsOnly)}
+                        {columnHeader}
```

Comment:
Yeah, I was originally planning to add props to the column header, but a) ended up going in a different direction and b) it didn't work anyways because react doesn't let you (good for it)